src/lib/eliom_request.client.ml

```diff
@@ -75,12 +75,21 @@ let xml_result x =
 let string_result x = x.XmlHttpRequest.content
 
 (*TODO: use Url.Current.set *)
-let redirect_get url = Dom_html.window##.location##.href := Js.string url
+let redirect_get ?window_name ?window_features url =
```

Comment:
Can you merge those two arguments into one (in a couple for instance) since calling this function with `window_name` at `None` discards any value provided for `window_features`?

Reply:
Yes but this would break uniformity/compatibility with previous versions ...